pkg/gui/custom_commands.go

```diff
@@ -118,16 +123,21 @@ func (gui *Gui) menuPrompt(prompt config.CustomCommandPrompt, promptResponses []
 	return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
 }
 
-func (gui *Gui) GenerateMenuCandidates(commandOutput string, filter string, format string) ([]string, error) {
-	candidates := []string{}
+func (gui *Gui) GenerateMenuCandidates(commandOutput, filter, valueFormat, labelFormat string) ([]CommandMenuEntry, error) {
+	candidates := []CommandMenuEntry{}
 	reg, err := regexp.Compile(filter)
 	if err != nil {
 		return candidates, gui.surfaceError(errors.New("unable to parse filter regex, error: " + err.Error()))
 	}
-	buff := bytes.NewBuffer(nil)
-	temp, err := template.New("format").Parse(format)
+	buffItem := bytes.NewBuffer(nil)
+	tempItem, err := template.New("format").Parse(valueFormat)
+	if err != nil {
+		return candidates, gui.surfaceError(errors.New("unable to parse item format, error: " + err.Error()))
+	}
+	buffDescr := bytes.NewBuffer(nil)
+	tempDescr, err := template.New("format").Parse(labelFormat)
```

Comment:
I think this is a better name
Note the change of desc**r** > desc, spell checker in the browser and in my editor complains about descr
```suggestion
	descBuff := bytes.NewBuffer(nil)
	descTemp, err := template.New("format").Parse(labelFormat)
```